lib/pages/magic_link/invitation_people_page.dart

```diff
@@ -0,0 +1,346 @@
+           child: Column(
+             mainAxisAlignment: MainAxisAlignment.spaceBetween,
+             children: [
+               Column(
+                 children: [
+                   Align(
+                       alignment: Alignment.topRight,
+                       child: GestureDetector(
+                         onTap: () => Navigator.of(context).pop(),
+                           child: Image.asset(imagePathCancel, width: 24, height: 24))),
+                   Text(AppLocalizations.of(context)?.generateNewLink ?? '',
+                       style: StylesConfig.commonTextStyle.copyWith(fontWeight: FontWeight.bold, fontSize: 20)),
+                   SizedBox(height: 12),
+                   Text(AppLocalizations.of(context)?.generateNewLinkSubtitle ?? '',
+                       style: StylesConfig.commonTextStyle.copyWith(fontSize: 15), textAlign: TextAlign.center),
+                   SizedBox(height: 16),
+                   _buildLinkField(),
+                   (state.status == InvitationStatus.generateLinkSuccess)
+                   ? Container(
+                     margin: const EdgeInsets.only(top: 8),
+                     child: Row(
+                       children: [
+                         Image.asset(imageValid, width: 14, height: 14),
+                         SizedBox(width: 8),
+                         Expanded(
+                           child: Text(AppLocalizations.of(context)?.newLinkGenerated ?? '',
+                            style: StylesConfig.commonTextStyle.copyWith(color: Color(0xff4bb34b), fontSize: 13),))
+                       ],
+                     ),
+                   )
+                   : SizedBox.shrink()
+                 ],
+               ),
+               Container(
+                 margin: const EdgeInsets.only(bottom: 12),
+                 child: ButtonTextBuilder(
+                     Key('button_generate_new_link'),
+                     onButtonClick: () => _handleClickOnButtonGenerateLink())
+                     .setWidth(double.infinity)
+                     .setHeight(50)
+                     .setText(AppLocalizations.of(context)?.generateNewLink ?? '')
+                     .setTextStyle(StylesConfig.commonTextStyle.copyWith(fontSize: 17, color: Colors.white, fontWeight: FontWeight.w500))
+                   .build(),
+               )
+             ],
+           ),
+         );
+      });
+  }
+
+  _handleClickOnButtonShareLink() async {
+    await Share.share(_urlNotifier.value);
+  }
+
+  _handleClickOnButtonInviteByEmail() {
+    NavigatorService.instance.navigateToInvitationPeopleEmail(_urlNotifier.value);
+  }
+
+  _handleClickOnLink() {
+    Clipboard.setData(new ClipboardData(text: _urlNotifier.value)).then((_){
+      ScaffoldMessenger.of(context).showSnackBar(
+        SnackBar(
+          backgroundColor: Colors.white,
+          margin: EdgeInsets.fromLTRB(10.0, 20.0, 10.0, MediaQuery.of(context).size.height - 110),
```

Comment:
Consider using Dim instead of MediaQuery.of(context).size.height

Reply:
Okay